lib/newrelic_ignore_downlevel_browsers.rb

```diff
@@ -6,10 +6,13 @@ def exclude_from_deprecated_browsers(script_tag)
       return '' if script_tag == ''
 
       # The following guard will prevent versions of IE <= 9 from processing
-      # the script_tag while exposing it to IE10+ and other browsers. (Note that
-      # IE10+ and other browser don't support conditional comments so the
-      # guard will does nothing in those cases.)
-      result = "<!--[if !IE]><!--> #{script_tag} <!--<![endif]-->"
+      # the script_tag while exposing it to IE10+ and other browsers.
+      # We then follow it with a script that will disable NewRelic on IE10, tho
+      # unlike the IE <=9 case, will still result in us loading the script
+      result = "<!--[if !IE]><!--> " +
+        "#{script_tag} " +
+        "<script>if (navigator.userAgent.match('MSIE 10.0;')) { delete window.NREUM; }</script>" +
+        "<!--<![endif]-->"
```

Comment:
seems reasonable. worst case, we start seeing new relic errors in IE10 again, right?